spec/src/main/asciidoc/repository.asciidoc

```diff
@@ -777,13 +777,15 @@ The Jakarta Data implementation automatically recognizes the query annotations i
 
 === Parameter-based automatic query methods
 
-An _automatic query method_ is an abstract method that either generates a query based on the parameters of the method. The method return type identifies the entity. For example: `E`, `Optional<E>`, `Page<E>`, or `List<E>`, where `E` is an entity class. Each parameter must either:
+An _automatic query method_ is an abstract method annoated using an _automatic query annotation_. Jakarta Data infers a query based on the parameters of the method. The method return type identifies the entity. For example: `E`, `Optional<E>`, `Page<E>`, or `List<E>`, where `E` is an entity class. Each parameter must either:
```

Comment:
Grrr should be "annotated".

Reply:
Github isn't allowing suggestions here, but this language is incorrect. Otavio, please change the beginning of the sentence,
"An _automatic query method_ is"
to instead be:
"A parameter-based _automatic query method_ is".
We need to be precise here, because Query by Method Name is also a type of automatic query method, and it does not fit the definition here.

Reply:
Done.